### PR TITLE
[plugin.video.drnu] 6.3.0

### PR DESCRIPTION
**Pull request by TermeHansen**
### Description
*Version 6.3.0 - 2024-02-25*
- Added login
- changed videoinfo to nexus api
- removed favourites and recent to use locale files

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by TermeHansen** [comment]
This commit is the actual difference from 6.2.0 (which is in the matrix branch here) to 6.3.0
https://github.com/xbmc-danish-addons/plugin.video.drnu/commit/9bf5def3c7a015db0d44fb0a76923a396fb1e69e

File: plugin.video.drnu/resources/lib/__pycache__/__init__.cpython-310.pyc
Comment (romanvm) [inline plugin.video.drnu/resources/lib/__pycache__/__init__.cpython-310.pyc:1]: Please remove all .pyc files from the PR and add them to .gitignore to avoid committing them to Git.

**Review by romanvm — CHANGES_REQUESTED** [review]
Please remove .pyc files

File: plugin.video.drnu/resources/lib/__pycache__/__init__.cpython-310.pyc
Comment (TermeHansen) [reply plugin.video.drnu/resources/lib/__pycache__/__init__.cpython-310.pyc:1]: @romanvm sorry my mistake, I forced push the amendment...

**Review by romanvm — APPROVED** [review]
Now LGTM

**Comment by TermeHansen** [comment]
@romanvm it is now waiting for another reviewer to merge or?

**Pull request merged**
